requirements-app.txt

```diff
@@ -6,6 +6,7 @@ Flask-Login==0.2.11
 Flask-WTF==0.12
 lxml==3.8.0
 
-git+https://github.com/alphagov/digitalmarketplace-utils.git@29.0.0#egg=digitalmarketplace-utils==29.0.0
+git+https://github.com/alphagov/digitalmarketplace-utils.git@31.2.1#egg=digitalmarketplace-utils==31.2.1
 git+https://github.com/alphagov/digitalmarketplace-content-loader.git@4.4.1#egg=digitalmarketplace-content-loader==4.4.1
 git+https://github.com/alphagov/digitalmarketplace-apiclient.git@14.0.0#egg=digitalmarketplace-apiclient==14.0.0
+git+https://github.com/alphagov/odfpy.git@ee4482a#egg=odfpy==1.3.6dev
```

Comment:
(Just an FYI for reviewers that we had to pull the `odfpy` in due to a dependency in the utils package.)